frontend/web/components/EditPermissions.tsx

```diff
@@ -27,16 +21,16 @@ import Switch from './Switch'
 import TabItem from './base/forms/TabItem'
 import Tabs from './base/forms/Tabs'
 import UserGroupList from './UserGroupList'
-import { PermissionLevel } from 'common/types/requests'
+import { PermissionLevel, Req } from 'common/types/requests'
```

Comment:
Reading it here, I'm wishing for a longer typed out version of `Req` since this seems a little too short hand in context.